docs/adr/0014-enforcing-resource-quotas.md

```diff
@@ -0,0 +1,44 @@
+# Enforcing of Resource Quotas
+
+Technical Story: [1][1], [2][2]
+
+## Context and Problem Statement
+Currently clusters being managed by Operate-First are not enforcing quotas on compute and storage resources. This means that anyone can use their `namespace` to use hog as many resources as they wish. We should use Kubernetes' [ResourceQuotas][3] to address this concern. Decisions for declaring quotas for resource `requests` and `limits` should be guided by monitoring current trends and usage via available dashboards. We will also need to add default [LimitRanges][4] so current workloads are not prevented from creating resources.
+
+## Considered Options
+
+The only option is to use [ResourceQuotas][3], what is subject to debate is how we organize/advertise these quotas in git.
+
+### 1. Tiered & Custom Resource quotas
+
+We identify a tier of standard `ResourceQuotas` that we distribute to each namespace. We will adhere to this tiered system as much as possible, but acknowledge the need for custom quotas, and allow for such exceptions on a need basis. `ResourceQuotas` can be included as `Kustomize` components
+
+### Pros:
+- User friendly. It's a lot easier to tell a new-comer to pick from a set of options then to specify specific requirements.
```

Comment:
I'd add one point here: consistency

Tiers ensure that all new-comers are treated the same and any custom request is treated more carefully.

Reply:
updated